### ISSUE TITLE
Should rule #9 be upgraded to a SHOULD

### ISSUE THREAD

**Issue by BurtHarris** [mojombo/semver#315]
Currently, rule 9 indicates that use of pre-release suffixes is optional, by using the word MAY.   

In real-world implementations, like npm, consumers often depend on publishers following rule 9.   It seems to me like the rule should be updated to use the word SHOULD in place of the first occurance of "MAY".

**Comment by Haacked** [comment]
Hmm, I can see how that is a bit confusing. The way I read it, it's "MAY" because not every version is a pre-release version. So a pre-release version MAY be denoted.

**Comment by BurtHarris** [comment]
OK, good point.   So perhaps rule nine should begin with "**Any** pre-release version **SHOULD** be denoted by appending a hyphen..."

**Comment by FichteFoll** [comment]
The way you phrased it requires "MUST" to be used.

**Comment by crazedsanity** [comment]
I agree that pre-releases must be distinguished from regular/non-pre-release versions.  Having "MAY" there seems indicative of the possibility that pre-releases might not be distinguishable as such.  So rule 9 would then change to:

> A pre-release version **MUST** be denoted by appending a hyphen...

Just wanted to (hopefully) clarify the wording to ensure we're all on the same page.

**Comment by BurtHarris** [comment]
I'd be happy enough with either version.

**Comment by tdpsk** [comment]
I agree that in practice, pre-release tags should be mandatory because users rely on them in the context of package repositories. Therefore I have created a corresponding pull request in #316, feel free to comment and discuss.

**Comment by FichteFoll** [comment]
>pre-release tags should be mandatory

This could be misunderstood, but the wording in the PR is clear.
